packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -205,6 +207,10 @@ const destructVmsPattern = pattern =>
         vms: destructPattern(pattern),
       }
 
+const isRetentionHigh = (settings, retention) =>
+  settings.getIn(['', 'fullInterval']) > RETENTION_LIMIT ||
```

Comment:
`||` -> `&&` (fullInterval: 51, retention: 1)

Reply:
If `fullInterval` >50 or `retention`> 50, we will have a long delta chain, that's why we add this warning.